content/docs/dvclive/get-started.md

```diff
@@ -1,166 +1,196 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning parameters, metrics and
-other metadata in simple file formats, which is fully compatible with DVC.
+other metadata in simple file formats, which is fully compatible with DVC, the
+[VS Code Extension](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc),
+and [Iterative Studio](https://studio.iterative.ai/).
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python script:
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+<toggle>
+<tab title="Keras">
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback()])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### (Optionally) Log parameters
+</tab>
 
-<toggle>
-<tab title="Single parameter">
+<tab title="Hugging Face">
 
 ```python
-live.log_param("num_classes", 10)
+# train.py
+from dvclive.huggingface import DvcLiveCallback
+
+. . .
+
+ trainer = Trainer(
+    model, args,
+    train_dataset=train_data,
+    eval_dataset=eval_data,
+    tokenizer=tokenizer,
+    compute_metrics=compute_metrics,
+)
+trainer.add_callback(DvcLiveCallback())
+trainer.train()
 ```
 
-See `Live.log_param()`.
-
 </tab>
-<tab title="Multiple parameters">
+<tab title="Pytorch Lightning">
 
 ```python
-params = {
-    "num_classes": 10,
-    "metrics": ["accuracy", "mae"],
-    "optimizer": "adam"
-}
-live.log_params(params)
-```
+# train.py
+from dvclive.lightning import DvcLiveLogger
 
-See `Live.log_params()`.
+. . .
+dvclive_logger = DvcLiveLogger()
+
+trainer = Trainer(logger=dvclive_logger)
+trainer.fit(model)
+```
 
 </tab>
-</toggle>
 
-### Log data
-
-<toggle>
-<tab title="Scalars">
+<tab title="Python API">
```

Comment:
Slightly unclear what this means: 1) All the samples are Python; 2) There's no "Python API" under ML Frameworks. Maybe link to https://dvc.org/doc/dvclive/api-reference/live for this one? Currently they all have the same comment under:

> Check the [ML Frameworks](https://dvc.org/doc/dvclive/api-reference/ml-frameworks) page for more details and other supported frameworks.